dns-records-office365/azuredeploy.json

```diff
@@ -15,11 +15,11 @@
 				"description": "Time to live value, in seconds"
 			}
 		},
-		"artifactsBaseUrl": {
+		"_artifactsLocation": {
```

Comment:
need to include the _artifactsLocationSasToken as well...  Everything else looks good.